diff --git a/Source/Drivers/PS1080/PS1080Console/PS1080Console.cpp b/Source/Drivers/PS1080/PS1080Console/PS1080Console.cpp
index e9e23ab..5635284 100644
--- a/Source/Drivers/PS1080/PS1080Console/PS1080Console.cpp
+++ b/Source/Drivers/PS1080/PS1080Console/PS1080Console.cpp
@@ -953,7 +953,7 @@ bool Download(openni::Device& Device, vector<string>& Command)
 		printf("Can't understand %s as id\n", Command[1].c_str());
 		return true;
 	}
-	XnParamFileData ParamFile = {nId, Command[2].c_str()};
+	XnParamFileData ParamFile = {(uint32_t) nId, Command[2].c_str()};
 
 	openni::Status rc = Device.getProperty(XN_MODULE_PROPERTY_FILE, &ParamFile);
 	if (rc != openni::STATUS_OK)
diff --git a/ThirdParty/PSCommon/BuildSystem/Platform.Arm b/ThirdParty/PSCommon/BuildSystem/Platform.Arm
index c4a112d..996f2f0 100644
--- a/ThirdParty/PSCommon/BuildSystem/Platform.Arm
+++ b/ThirdParty/PSCommon/BuildSystem/Platform.Arm
@@ -1,7 +1,7 @@
 ifeq "$(CFG)" "Release"
 
     # Hardware specifying flags
-    CFLAGS += -march=armv7-a -mtune=cortex-a9 -mfpu=neon -mfloat-abi=softfp #-mcpu=cortex-a8
+    CFLAGS += -march=armv7-a -mtune=cortex-a9 -mfpu=neon #-mcpu=cortex-a8
 
     # Optimization level, minus currently buggy optimizing methods (which break bit-exact)
     CFLAGS += -O3 -fno-tree-pre -fno-strict-aliasing
